src/store/authors/thunk.js

```diff
@@ -0,0 +1,29 @@
+import { REACT_APP_BASE_URL } from '../../env-config';
+import { actionCreators } from './actionCreators';
+
+export async function thunk_getAuthors(dispatch) {
```

Comment:
I'd not recommend to use thinks like this, even if function doesn't have any arguments. Please, create a wrapper function  (similar to `addAuthorThunk `, but without any arguments) and dispatch it like `dispatch(getAuthorsThunk())`